kudu/Dockerfile

```diff
@@ -76,6 +76,15 @@ ENV PATH=$PATH:/opt/nodejs/9/bin
 
 EXPOSE 8181
 
+# Install ClamAV
+RUN apt-get install -y clamav
+RUN freshclam
```

Comment:
Move this part to the `# Install dependencies` part

Reply:
Done